tonic_validate/metrics/answer_consistency_metric.py

```diff
@@ -17,6 +17,13 @@
 class AnswerConsistencyMetric(Metric):
     name: str = "answer_consistency"
 
+    def __init__(self):
+        """
+        Metric that checks whether there is information in the LLM answer that does not come from the context.
```

Comment:
Metric that checks whether the LLM answer contains information that does not come from the context.